src/components/Description/index.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import './style.less';
+import heart from '../../assets/heart.svg';
+import DescriptionImg from '../../assets/description_image.png';
+
+const Description: React.FC = () => (
+  <div className="description">
+    <div className="description-content">
+      <div className="description-text-container">
+        <div className="description-text">
```

Comment:
Try reducing the amount of divs you need here. You can actually just use 1-2 divs with padding and `display: flex`, `align-items: center`, and `justify-content: center`

Reply:
(^･o･^)ﾉ” 
okie
![image](https://user-images.githubusercontent.com/32044772/89116165-dafbfa00-d445-11ea-8a43-2b7b528441f9.png)